axum-extra/src/response/file_stream.rs

```diff
@@ -0,0 +1,638 @@
+use axum::{
+    body,
+    response::{IntoResponse, Response},
+    BoxError,
+};
+use bytes::Bytes;
+use futures_util::{Stream, TryStream};
+use http::{header, StatusCode};
+use std::{io, path::Path};
+use tokio::{
+    fs::File,
+    io::{AsyncReadExt, AsyncSeekExt},
+};
+use tokio_util::io::ReaderStream;
+
+/// Alias for `tokio_util::io::ReaderStream<File>`.
+pub type AsyncReaderStream = ReaderStream<File>;
```

Comment:
Why the type alias? `ReaderStream<File>` is not a very complex type.

Reply:
> Why the type alias? `ReaderStream<File>` is not a very complex type.

It's indeed unnecessary, I just removed it for the sake of a shorter generic, and it's fine to remove it, I removed the